packages/iris-grid/src/AdvancedFilterCreator.tsx

```diff
@@ -565,6 +568,7 @@ class AdvancedFilterCreator extends PureComponent<
                 onClick={this.handleSortDown}
                 icon={dhSortAmountDown}
                 tooltip={`Sort ${column.name} Descending`}
+                disabled={!column.isSortable}
```

Comment:
@dsmmcken I wasn't sure about in Advanced Filters - are we just disabling these buttons? Or should it say "Not sortable" with that icon?

Reply:
I recommended disable, but could also change the tooltip to "Column is not sortable"